handlers/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: Restart gmrender-resurrect service
+  service:
+    name: gmrender-resurrect
+    daemon_reload: true
```

Comment:
this is not necessary for all invocations of the handler, you may want to change this.